sap-xmake-build.sh

```diff
@@ -0,0 +1,115 @@
+#!/bin/bash -l
+echo "Inside file -- sap-xmake-build.sh --"
+
+# find this script and establish base directory
+curr_dir=`dirname $0`
+MY_DIR=`cd $curr_dir; pwd`
+echo "[INFO] Executing in ${MY_DIR}"
+
+PACKAGE_NAME=${PACKAGE_NAME:-"scala-pickling"}
+PACKAGE_BRANCH=${PACKAGE_BRANCH:-"0.11.x"}
+BUILD_BRANCH=${BUILD_BRANCH:-"v0.11.x-M1_2.11"}
+ALTISCALE_RELEASE="${ALTISCALE_RELEASE:-5.0.0}"
+
+mkdir -p hack_target
+pushd hack_target
+rm -rf *.jar
```

Comment:
I wonder what does this do here?

Reply:
This is not required, it was there to save jar dependencies fetched via wget. We do not need this anymore. Will remove it in next commit.